### PR TITLE
Momentum UI

### PR DESCRIPTION
**Pull request by mokandre**

### CODE REVIEW

File: packages/node_modules/@ciscospark/react-component-activity-item/src/index.js [hunk 1]
```diff
@@ -80,16 +82,27 @@ function ActivityItem(props) {
   } = props;
 
   let itemComponent = '';
+  let shouldRenderAdaptiveCard;
```
Comment (Saumyacd) [inline packages/node_modules/@ciscospark/react-component-activity-item/src/index.js]: Let's have a default value for shouldRenderAdaptiveCard, as value assignment to it is conditional.

File: packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  internalRender() {
+    this._selectElement = document.createElement('div');
+    if (!this.isMultiSelect) {
+      const defaultChoice = this.choices.find((choice) => this.defaultValue === choice.value) || null;
+
+      this.value = this.defaultValue || null;
+
+      if (this.isCompact) {
+        // Render Select box
+        let defaultOptionText = 'Choose one';
```
Comment (Saumyacd) [inline packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js]: could we get this value from a constant file

File: packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+        </RadioGroup>
+      );
+
+      renderCardElement(jsx, this._selectElement, this.addChildNode);
+
+      return this._selectElement;
+    }
+
+    // Render Checkboxes
+    const defaultOptionText = this.title || this.placeholder || 'Choose';
```
Comment (Saumyacd) [inline packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js]: could we get this value ('Choose') from constant file?

File: packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js (hunk 2 above)
Comment (mokandre) [reply packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js]: Sure , I will move it to constant file

File: packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js (hunk 3 above)
Comment (mokandre) [reply packages/node_modules/@ciscospark/react-component-md-choiceset-input/src/index.js]: Same as above

**Pull request closed without merging**